usagov/vote-gov-drupal
# Feature/vote 2067 breadcrumb space

## Jira ticket

[Vote-2067](https://cm-jira.usa.gov/browse/VOTE-2067)

## Description

Adding style to the div wrapping the breadcrumb block to add a margin top style to the following element. 

Also updated the `page-title` template to removed the div and just make it into an `h1` and update the styles for the component.  

Side by side of computed styles:
![Screenshot 2024-06-12 at 9 17 56 AM](https://github.com/usagov/vote-gov-drupal/assets/88721460/6d206705-f8ba-4309-9711-d63af933036e)


## Deployment and testing

### Post-deploy steps

1. run `lando retune` and cd into the `votegov` theme and run `npm run build`

### QA/Testing instructions

1. visit the following pages `voter guide`, `about page`, `state selection page`, `accessibility page`, `site map` and inspect the heading element and check that the margin top has 5rem/80px
2. also add `{{ drupal_entity('block', 'votegov_breadcrumbs') }}` above the `vote-hero__heading` element in the hero file and check that the same styles are applied to the hero element on the home page as well.

## Checklist for the Developer




## Checklist for the Peer Reviewers
